src/alpaca_eval/models_configs/llama-2-7b-chat-hf/prompt.txt

```diff
@@ -0,0 +1,5 @@
+[INST] <<SYS>>
```

Comment:
@lxuechen  this is what I gathered from https://github.com/facebookresearch/llama/blob/main/llama/generation.py#L212

maybe worth double checking. Unclear which of those special tokens I need to add

Reply:
can you check that the `input_ids` after tokenization match?

Reply:
didn't but I checked string match. but we should careful with prompts, both hugging face and llmsys seem to be doing it wrong (different mistakes)
- [hugging face](https://huggingface.co/spaces/ysharma/Explore_llamav2_with_TGI/blob/main/app.py) has one additional space after \n<</SYS>>\n\n
- [lmsys](https://github.com/lm-sys/FastChat/pull/1995/files#diff-21c1e357f53a0a6d5d5a812e75710d0e3b013c889373a7c6c5019c491b2c05f1R847) is missing a space in front of "[/INST]"